### PR TITLE
don't reassess obs on global non-obscuring status creation/deletion

### PR DESCRIPTION
**Pull request by loarie**
when global non-obscuring conservation statuses are created or destroyed, no need to reassess observation geoprivacies

### CODE REVIEW

**Comment by loarie** [comment]
@kueda ok to merge this in?

File: app/models/conservation_status.rb [hunk 1]
```diff
@@ -124,6 +124,8 @@ def normalize_geoprivacy
 
   def update_observation_geoprivacies
     return true if skip_update_observation_geoprivacies
+    # Don't reassess obs for newly created or destroyed non-obscuring global statuses
+    return true if ( saved_change_to_id? || destroyed? ) && geoprivacy == Observation::OPEN && place_id.nil?
```
Comment (kueda) [inline app/models/conservation_status.rb:128]: On staging there are 55991 ConservationStatus records where the `geoprivacy` column is `null` not `open`, so I think you're going to want to check for that here.

File: spec/models/conservation_status_spec.rb [hunk 2]
```diff
@@ -57,6 +57,26 @@
...
+      cs = ConservationStatus.make!( taxon: species )
+      jobs = Delayed::Job.where("created_at >= ?", stamp)
+      expect(jobs.select{|j| j.handler =~ /reassess_coordinates_for_observations_of/m}).not_to be_blank
+    end
+
+    it "should not reassess observations of taxon if open and global" do
+      species = Taxon.make!( rank: Taxon::SPECIES )
+      Delayed::Job.delete_all
+      stamp = Time.now
+      cs = ConservationStatus.make!( taxon: species, geoprivacy: "Open" )
```
Comment (kueda) [inline spec/models/conservation_status_spec.rb:74]: Use `Observation::OPEN` here instead of the string literal. Testing against `"Open"` is testing against a value that doesn't exist in reality. You should probably also test the situation where geoprivacy is nil.

**Review by kueda — CHANGES_REQUESTED** [review]

**Review by kueda — APPROVED** [review]
